docs/dev/table/blink.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Configuration"
+nav-id: "config"
+nav-parent_id: tableapi
+nav-pos: 4
```

Comment:
Change the `nav-pos` to make config page to be the last one under "Table API & SQL"?